roles/network/templates/hostapd/hostapd.conf.j2

```diff
@@ -1,7 +1,10 @@
 # Basic configuration
 
-interface={% if iiab_wireless_lan_iface is defined %}{{ iiab_wireless_lan_iface }}{% endif %}
-
+{%if wifi_up_down  %}
+interface=ap0
+{% else %}
+interface={{ iiab_wireless_lan_iface }}
```

Comment:
wifi only installs would not have iiab_wireless_lan_iface present I would suggest using discovered_wireless_iface